src/ResourceManager/ServiceBus/Commands.ServiceBus.Test/ScenarioTests/ServiceBusQueueTests.ps1

```diff
@@ -46,6 +46,7 @@ function ServiceBusQueueTests
 	$resultGetQueue.DeadLetteringOnMessageExpiration = $True
 	$resultGetQueue.MaxDeliveryCount = 5
 	$resultGetQueue.MaxSizeInMegabytes = 1024
+	$resultGetQueue.EnableBatchedOperations = $True
```

Comment:
@v-Ajnava are there any tests that cover the new switch parameters being used rather than the property being set on the object?

Reply:
its used to set the batchoperations service side. nothing no client side besides setting it.